extension.ts

```diff
@@ -346,12 +366,17 @@ const findVSCodeRecentItems = (
 
     const storage = JSON.parse(ByteArray.toString(contents));
 
-    const recentWorkspaceURIs: string[] =
+    const recentWorkspaceItems: ReadonlyArray<WorkspaceItem> =
```

Comment:
You've said that you don't know how this looks like, so shouldn't we rather use `unknown` here and perform a series of explicit type checks, perhaps with custom predicates?

Reply:
We could perhaps use `string | Record<string, unknown>`.